modules/preview-react/text/stories/examples/Label.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import {Box} from '@workday/canvas-kit-react';
+import {Label as LabelText} from '@workday/canvas-kit-preview-react/text';
+
+export const Label = () => (
+  <>
+    <Box marginBottom="s">
+      <LabelText size="large">Label</LabelText>
```

Comment:
```suggestion
      <LabelText marginBottom="s" size="large">Label</LabelText>
```

Reply:
Hm. Why did i even use `Box`? Thank for catching it.